docker-compose.yml

```diff
@@ -39,6 +39,16 @@ services:
     command: --setParameter enableTestCommands=1
     container_name: dance_mongodb
     restart: unless-stopped
+    ulimits:
+      nofile:
+        soft: 64000
+        hard: 64000
+      nproc: 64000
+    deploy:
+      resources:
+        limits:
+          cpus: '2'
+          memory: 7G
```

Comment:
Matches the runner on CI ubuntu-22.04 and sets ulimits per the [recommendation](https://www.mongodb.com/docs/upcoming/reference/ulimit/).

Reply:
Could you add this in a comment please. :hugs:

Reply:
I don't think we should _slow down_ our tests to match slow CI

Reply:
I agree, do you propose using a larger runner on CI?

Reply:
Maybe. But that's not related to this PR, I guess (and we probably should update the PR title).

Reply:
Cool!